media/js/base/site.js

```diff
@@ -64,10 +64,7 @@
             if (pf.indexOf("Mac") !== -1) {
                 return 'oldmac';
             }
-            if (pf === '' &&
-                    /Firefox/.test(ua) &&
-                    /Mobile|Tablet/.test(ua) &&
-                    !(/Android/.test(ua))) {
+            if (pf === '' && /Firefox/.test(ua)) {
```

Comment:
Based on the UA patterns in https://developer.mozilla.org/en-US/docs/Web/HTTP/Gecko_user_agent_string_reference#Firefox_OS and https://bugzilla.mozilla.org/show_bug.cgi?id=1191136#c0 the following regex should allow us to match the FxOS UA exclusively:

`/^Mozilla\/([0-9.])+ \(.*\) Gecko\/([0-9.])+ Firefox\/([0-9.])+$/`